### PR TITLE
Took out map and compass under uncommon village loot

### PR DESCRIPTION
**Pull request by Rebel-Leader**

### CODE REVIEW

File: config/Wasteland/ChestLoot.cfg [hunk 1]
```diff
@@ -163,9 +163,7 @@
         minecraft:pumpkin_pie,1,1
         minecraft:red_flower,1,1
         minecraft:iron_ingot,2,1
-        minecraft:compass,1,1
```
Comment (chiqors) [inline config/Wasteland/ChestLoot.cfg:166]: we don't need that?

File: config/Wasteland/ChestLoot.cfg [hunk 2]
```diff
@@ -163,9 +163,7 @@
...
         NuclearCraft:fishAndRicecake,1,1
-        minecraft:map,1,1
```
Comment (chiqors) [inline config/Wasteland/ChestLoot.cfg:168]: same thing like compass?

File: config/Wasteland/ChestLoot.cfg (hunk 1 above)
Comment (Rebel-Leader) [reply config/Wasteland/ChestLoot.cfg:166]: with journey map its pretty useless. even if it is a crafting recipe item its easy to make.

File: config/Wasteland/ChestLoot.cfg (hunk 2 above)
Comment (Rebel-Leader) [reply config/Wasteland/ChestLoot.cfg:168]: yes

**Review by chiqors — APPROVED** [review]
Done

**Pull request merged**